insalan/cms/tests.py

```diff
@@ -3,27 +3,34 @@
 from django.core.exceptions import ValidationError
 from django.utils.translation import gettext_lazy as _
 
-from insalan.cms.models import (
-    Constant,
-    Content
-)
+from insalan.cms.models import Constant, Content
 
 
 class ContentTestCase(TestCase):
     def test_content_with_constants(self):
-        """ test that we can create a content using constant """
+        """test that we can create a content using constant"""
         Constant.objects.create(name="a", value="je suis la valeur a")
         Constant.objects.create(name="b", value="je suis la valeur b")
-        Content.objects.create(name="content_a", content="j'appelle la constante ${a} et la constante ${b}")
+        Content.objects.create(
+            name="content_a", content="j'appelle la constante ${a} et la constante ${b}"
+        )
 
     def test_content_with_unknow_constants(self):
```

Comment:
I think the following three methods are those @Lugrim wanted you to document a tiny bit, even just for the sake of consistency.

Reply:
fixed by [092dd48](https://github.com/InsaLan/backend-insalan.fr/pull/92/commits/092dd488d5ec8dfa7a524c08747fc6d20c6e2dc2)